pkg/controller/import-controller.go

```diff
@@ -548,6 +548,7 @@ func (r *ImportReconciler) createImporterPod(pvc *corev1.PersistentVolumeClaim)
 	pod, err := createImporterPod(context.TODO(), r.log, r.client, podArgs, r.installerLabels)
 	// Check if pod has failed and, in that case, record an event with the error
 	if podErr := cc.HandleFailedPod(err, pvc.Annotations[cc.AnnImportPod], pvc, r.recorder, r.client); podErr != nil {
+		r.log.V(1).Error(err, "failed creating pod", "pod.Name", pod.Name)
```

Comment:
ugh seems like log for cdi-controller was rotated. got bit by that too https://github.com/kubevirt/containerized-data-importer/pull/3795/commits/8d6167b833e4d48c06576c000774359395e37d8b